Source/Turbo/Navigator/Navigator.swift

```diff
@@ -111,6 +111,14 @@ public class Navigator {
     public func appDidEnterBackground() {
         appInBackground = true
     }
+    
+    public func activeWebview() -> WKWebView {
```

Comment:
I'd prefer this to be a computed property, like `activeNavigationController`:

```swift
var activeWebView: WKWebView {
    if activeNavigationController == rootViewController {
        return session.webView
    }
    return modalSession.webView
}
```

Reply:
Ah I see, good point. I'll update it, so it's aligned with the others public properties.

Reply:
[minor] 

```swift
var activeWebView: WKWebView {
   return activeNavigationController == rootViewController ? session.webView : modalSession.webView
}